api_app/analyzers_manager/file_analyzers/elf_info.py

```diff
@@ -0,0 +1,36 @@
+# This file is a part of IntelOwl https://github.com/intelowlproject/IntelOwl
+# See the file 'LICENSE' for copying permission.
+
+import logging
+
+from elftools.common.exceptions import ELFError
+from elftools.elf.elffile import ELFFile
+from telfhash import telfhash
+
+from api_app.analyzers_manager.classes import FileAnalyzer
+
+logger = logging.getLogger(__name__)
+
+
+class ELFInfo(FileAnalyzer):
+    def run(self):
+        results = {}
+        try:
+            with open(self.filepath, "rb") as file:
+                elf = ELFFile(file)
+            if elf is None:
+                raise ELFError("Not an ELF file")
+            results["telf"] = telfhash(self.filepath)[0]
+            results["telf"].pop("file", None)
```

Comment:
why do you remove this key?

Reply:
This tells the name of the temporary file created on the server. That's just a random name and hence irrelevant to the users.